### PR TITLE
[Kotlin Migration] TimeUtils

### PR DESCRIPTION
**Pull request by puranjayK**
[Kotlin Migration] TimeUtils
package com.ichi2.libanki.utils

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/utils/TimeUtils.kt [hunk 1]
```diff
@@ -13,17 +13,13 @@
...
-import androidx.annotation.NonNull;
-
-public class TimeUtils {
-    @NonNull
-    public static String getTimestamp(@NonNull Time time) {
-        return new SimpleDateFormat("yyyyMMddHHmmss", Locale.US).format(time.getCurrentDate());
+object TimeUtils {
+    fun getTimestamp(time: Time): String {
+        return SimpleDateFormat("yyyyMMddHHmmss", Locale.US).format(time.currentDate)
     }
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/libanki/utils/TimeUtils.kt:24]: I am not good at Kotlin, yet. We are converting a `public static` method to Kotlin though, and I thought we needed to do JvmStatic and a companion object for java interoperability, like so:


```suggestion
   @JvmStatic
   companion object {
      fun getTimestamp(time: Time): String {
          return SimpleDateFormat("yyyyMMddHHmmss", Locale.US).format(time.currentDate)
      }
    }
```

But it did pass compilation and unit tests :thinking: 

I must be missing something!

Comment (david-allison) [reply AnkiDroid/src/main/java/com/ichi2/libanki/utils/TimeUtils.kt:24]: All the callers are Kotlin, so it's fine without the `@JvmStatic`

**Review by david-allison — APPROVED** [review]
LGTM

File: AnkiDroid/src/main/java/com/ichi2/libanki/utils/TimeUtils.kt (hunk 1 above)
Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/libanki/utils/TimeUtils.kt:24]: Oh my - you mean we are far enough along in the conversion that for at least this case we don't need to worry about interop with Java :exploding_head: :laughing:. Awesome.
I guess that means there will be a big pruning of JvmStatic and companion objects later!
Thanks for explaining - obvious in hindsight but I guess I couldn't believe it.

**Review by mikehardy — APPROVED** [review]

**Comment by mikehardy** [comment]
Fantastic, thanks @puranjayK

**Pull request merged**